samples/03_org_administrators/validate_user_profiles.ipynb

```diff
@@ -0,0 +1,304 @@
+{
```

Comment:
<p>Hi Manushi, thanks for adding this notebook. I think this serves as a great tutorial for users who want to create a list or a DataFrame that contains all the user profiles. A minor suggestion I might have, is that now users can see False/True results, if there a way to have them view more content to user accounts, e.g. when access is True, what kind of access this user is granted? Also, is there a way to change the thumbnail or descriptions immediately? I believe users will have this confusion soon after this cell. Please do not hesitate to let me know if you think otherwise. Thanks!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1565/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
Thanks for the great feedback @CMPeng ! I updated that dataframe to also show access type.

The idea to change thumbnail and description is pretty good, but then I'm not sure if we will be distracting the user from the main flow of checking whether the org users have minimum attributes needed to qualify as a valid user, instead of a bot user. 
How about I add the steps to update thumbnail + description in this [accessing-and-managing-users](https://github.com/Esri/arcgis-python-api/blob/master/guide/03-the-gis/accessing-and-managing-users.ipynb) guide?

Reply:
"How about I add the steps to update thumbnail + description in this [accessing-and-managing-users](https://github.com/Esri/arcgis-python-api/blob/master/guide/03-the-gis/accessing-and-managing-users.ipynb) guide?"
 - Manushi, for the question listed above, sure we can do it in another guide :)